tools/toolchain/install_cp2k_toolchain.sh

```diff
@@ -302,7 +304,8 @@ with_spglib="__INSTALL__"
 with_hdf5="__INSTALL__"
 with_elpa="__INSTALL__"
 with_libvdwxc="__INSTALL__"
-
+with_spfft=__INSTALL__
+#with_cosma="__DONTUSE__"
```

Comment:
you probably don't want this commented, right? since the cosma script sourcing is commented out and the script is missing

Reply:
Well, I will remove it and add it when I am back in Europe. I do not want to test this while away.